### PR TITLE
fix: Added headers for cors in sales page

### LINKED ISSUE #8021: Sales admin page of eventyay not accessible
The sales admin page of eventyay gives an error message and is not accessible.

Compare https://eventyay.com/admin/sales/live

Screenshot from 2021-11-03 14-30-37

https://test.eventyay.com/admin/sales/live

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #8021 

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by codedsun** [comment]
Dont merge until approved

**Review by pc-beast — CHANGES_REQUESTED** [review]
Doesn't Work. I am not sure why it suddenly started throwing cors error.

**Pull request closed without merging**
